core/services/relay/evm/evm.go

```diff
@@ -319,7 +321,7 @@ func newContractTransmitter(lggr logger.Logger, rargs relaytypes.RelayArgs, tran
 		if sendingKeysLength > 1 && s == effectiveTransmitterAddress.String() {
 			return nil, errors.New("the transmitter is a local sending key with transaction forwarding enabled")
 		}
-		if err := ethKeystore.CheckEnabled(common.HexToAddress(s), configWatcher.chain.Config().ChainID()); err != nil {
+		if err := ethKeystore.CheckEnabled(evmtypes.NewAddress(common.HexToAddress(s)), configWatcher.chain.Config().ChainID()); err != nil {
```

Comment:
just a thought since this conversion happens quite a bit: might be useful to have a `evmtypes.HexToAddress` function

Reply:
Yes, that's a good idea. I had thought about it, but got lost in so many changes. Done.